tracing/plugin/otlp.go

```diff
@@ -160,7 +160,11 @@ func newTracer(ic *plugin.InitContext) (io.Closer, error) {
 	for id, pctx := range ls {
 		p, err := pctx.Instance()
 		if err != nil {
-			log.G(ctx).WithError(err).Errorf("failed to initialize a tracing processor %q", id)
+			if plugin.IsSkipPlugin(err) {
+				log.G(ctx).WithError(err).Infof("skip to initialize a tracing processor %q...", id)
```

Comment:
minor wording suggestion for readability: "skipping tracing processor initialization (no tracing plugin)"

Reply:
Updated